packages/aws-s3/aws-s3.1.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann"
+license: "BSD-3"
+homepage: "https://github.com/andersfugmann/aws-s3"
+dev-repo: "git+https://github.com/andersfugmann/aws-s3"
+bug-reports: "https://github.com/andersfugmann/aws-s3/issues"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" { build }
+  "core" { >= "0.9" }
+  "async"
+  "async_extended"
+  "ounit"
```

Comment:
Should this be a test dependency perhaps?

Reply:
Currently unit tests reside in the same file as the module implementation in order to have access to internal functions that may not be exposed by the module signature - so unfortunately for now it cannot be a test dependency only.